src/Workflow/Event.php

```diff
@@ -1,32 +1,28 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: gauthier
- * Date: 27/02/2018
- * Time: 15:40
- */
 
-namespace ObjectivePHP\Application\Exception;
+namespace ObjectivePHP\Application\Workflow;
 
-
-abstract class Workflow
+/**
+ * Interface Event
+ * @package ObjectivePHP\Application\Workflow
+ */
+interface Event
```

Comment:
I think this constants should be ApplicationInterface constants.

Reply:
It currently was in an Abstract class called Workflow that wasn't working, I just changed it to an Interface. But constant definition was not my main objective here :) Feel free to change it :)